### PR TITLE
v0.28.0 changelog blog post

### PR DESCRIPTION
**Pull request by cullywakelin**
Migrated content from Notion.

### CODE REVIEW

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+<!--truncate-->
+
+## What is included in this changelog?
+
+This changelog includes all of the changes since Temporal forked from Cadence.
+
+## What has changed?
+
+- **Temporal v0.27.0-**🅰️  ➡️ &nbsp**v0.28.0-**🅰️ : The software version has bumped.
```
Comment (rylandg) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:23]: Can you remove the -🅰️  and just put the word Alpha. Right now it looks like its an actual part of the version string

**Comment by cullywakelin** [comment]
> Overall looks great:
> 
> 1. Please replace the emoji arrows with actual arrows ->
> 2. Version shouldn't have 🅰️, just write Alpha or something
> 3. I think this is a good opportunity to start the post with a nice intro. Either introduce yourself or just talk a bit about the nature of this update.

1 - Done
2 - Done
3 - Done LMK what you think.

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+title: Temporal v0.28.0 changelog
+author: Cully Wakelin
+author_title: SDE Technical Writer
+author_image_url: https://avatars2.githubusercontent.com/u/34380806?s=400&u=5cd38b5e4416a5d10cdf9ebd386eec1d02f0b067&v=4
+release_version: v0.28.0
+---
+
+<!--truncate-->
+
+Dear Temporal users,
```
Comment (rylandg) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:17]: Too formal, this isn't a letter to the editor :) 

Maybe, "Hey, I'm Cully and I'm...".

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+Today is an exciting day for us as Temporal is releasing its newest version, v0.28.0, which  brings us closer to a stable production ready release. Thank you to all of the users who have helped us iterate and move the product forward.
```
Comment (rylandg) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:19]: "as Temporal is releasing", say this from a position of beloning. Right now it sounds like you're not part of the company.

"which  brings us clos"

an extra space between which.

also I would reword to say

"which is our first major step towards a production ready release. 

"Thank you to all of"

I want to thank all of the users

Comment (cullywakelin) [reply blog/2020-07-31-temporal-v0.28.0-changelog.md:19]: > "as Temporal is releasing", say this from a position of beloning. Right now it sounds like you're not part of the company.

"Today is an exciting day for us, as Temporal is releasing..." ?? "us" is used in the first clause of the sentence.

> "which brings us clos"
 Fixed.

> also I would reword to say
> "which is our first major step towards a production ready release.

Its not - its ONE more step closer - it could be argued that any of the previous releases were the FIRST major step... 

> "Thank you to all of"
> I want to thank all of the users - 

These are not my changes. I am drafting and publishing this on behalf of the company and the engineers. So, I prefer to keep this generally inclusive "We thank all of the users". I will be personal in chats, and discourse posts, or maybe specific blog posts. This doesn't seem like the right scenario to show this type of ownership.

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Today is an exciting day for us, as Temporal is releasing its newest version, v0.28.0, which brings us closer to a stable production ready release. We thank all of the users who have helped us iterate and move the product forward.
+
+This changelog includes all of the changes since Temporal forked from Cadence, and should help users who are interested in upgrading.
+
+## What has changed?
+
+- **Cadence Oct 2019** → **Temporal v0.28.0 Alpha**: The software version has bumped.
+- **TChannel/Thrift** → **gRPC/Protobuf**:
+    - Server Thrift API → Protobuf based Go API (go.temporal.io/api).
+    - All communication over the wire can be encrypted via [TLS](https://docs.temporal.io/docs/configuring-temporal-server#tls). Note: Support for TLS in tctl and temporal-web is still in-progress.
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:26]:
```suggestion
    - All communication over the wire can be encrypted via [TLS](https://docs.temporal.io/docs/configuring-temporal-server#tls). Note: Support for TLS in `tctl` and `temporal-web` is still in-progress.
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    - Thrift errors have been converted to custom service errors.
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:27]:
```suggestion
    - Thrift errors have been converted to custom service errors backed up by standart [gRPC error codes](https://pkg.go.dev/google.golang.org/grpc/codes).
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+<!--truncate-->
+
+Today is an exciting day for us, as Temporal is releasing its newest version, v0.28.0, which brings us closer to a stable production ready release. We thank all of the users who have helped us iterate and move the product forward.
+
+This changelog includes all of the changes since Temporal forked from Cadence, and should help users who are interested in upgrading.
+
+## What has changed?
+
+- **Cadence Oct 2019** → **Temporal v0.28.0 Alpha**: The software version has bumped.
+- **TChannel/Thrift** → **gRPC/Protobuf**:
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:24]: I would put serializator before transport:
```suggestion
- **Thrift/TChannel** → **Protobuf/gRPC**:
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+    - Removed Uber YARPC library.
+- **Helm charts** ☸️  : [Deploy Temporal server to a Kubernetes Cluster](https://github.com/temporalio/helm-charts).
+- **Entity renames** 🏷️ :
+    - `ArchivalStatus` → `ArchivalState`
+    - `Decider` → `Workflow`
+    - `Decision` → `Command`
+    - `DecisionTask` → `WorkflowTask`
+    - `Domain` → `Namespace`
+    - `NamespaceStatus` → `NamespaceState`
+    - `TaskList` → `TaskQueue`
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:38]: Let's sort these by importance, not abc.
```suggestion
    - `Decision` → `Command`
    - `DecisionTask` → `WorkflowTask`
    - `Decider` → `Workflow`
    - `Domain` → `Namespace`
    - `TaskList` → `TaskQueue`
    - `NamespaceStatus` → `NamespaceState`
    - `ArchivalStatus` → `ArchivalState`
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+## What has changed?
+
+- **Cadence Oct 2019** → **Temporal v0.28.0 Alpha**: The software version has bumped.
+- **TChannel/Thrift** → **gRPC/Protobuf**:
+    - Server Thrift API → Protobuf based Go API (go.temporal.io/api).
+    - All communication over the wire can be encrypted via [TLS](https://docs.temporal.io/docs/configuring-temporal-server#tls). Note: Support for TLS in tctl and temporal-web is still in-progress.
+    - Thrift errors have been converted to custom service errors.
+    - Standard gRPC health service is used for health checks.
+    - Removed Uber YARPC library.
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:29]: I think it worth to add between 28-29:
 - Standart protobuf types are used for times and durations.

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+- **Entity renames** 🏷️ :
+    - `ArchivalStatus` → `ArchivalState`
+    - `Decider` → `Workflow`
+    - `Decision` → `Command`
+    - `DecisionTask` → `WorkflowTask`
+    - `Domain` → `Namespace`
+    - `NamespaceStatus` → `NamespaceState`
+    - `TaskList` → `TaskQueue`
+- **Import path rename** 📦 : go.uber.org/cadence → go.temporal.io/sdk
+- **Workflow timeout refactors** ⌛:
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:40]: - ** Attempts counters start from 1 **

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+        - List Tasks ([#429](https://github.com/temporalio/temporal/pull/429)/[#433](https://github.com/temporalio/temporal/pull/433)): `shard list_tasks`
+        - Describe Task ([#408](https://github.com/temporalio/temporal/pull/408), [#416](https://github.com/temporalio/temporal/pull/416), [#412](https://github.com/temporalio/temporal/pull/412)): `"shard describe_task --task_type [timer, replication, transfer] ..."` / `"dt"`
+        - List Cluster memberships([#423](https://github.com/temporalio/temporal/pull/423/files)): `"membership list_db"`
+        - List Ringpop members ([#426](https://github.com/temporalio/temporal/pull/426)): `"membership list_gossip"`
+        - List Replication DLQ Tasks ([#456](https://github.com/temporalio/temporal/pull/456)): `"kafka list_dlq"`
+    - Admin Command improvements:
+        - Added filtering of Tasks from TaskQueue based on Workflow and run Ids  ([#462](https://github.com/temporalio/temporal/pull/462))
+        - Consistency to CLI admin command names ([#427](https://github.com/temporalio/temporal/pull/427))
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:62]: Is it in English?

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+        - Parameter types changed from enum to string ([#445](https://github.com/temporalio/temporal/pull/445), [#447](https://github.com/temporalio/temporal/pull/447))
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:63]:
```suggestion
        - Enum parameter flags changed from int to string ([#445](https://github.com/temporalio/temporal/pull/445), [#447](https://github.com/temporalio/temporal/pull/447))
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+        - Table-view support for CLI list commands ([#444](https://github.com/temporalio/temporal/pull/444))
+    - Fixes
+        - Admin cluster command for adding search attributes to be indexed ([#514](https://github.com/temporalio/temporal/pull/514))
+        - Non-string heartbeat details fail to display and only the first heartbeat detail shows up in CLI ([#525](https://github.com/temporalio/temporal/pull/525))
+        - Printing Query CLI command results ([#542](https://github.com/temporalio/temporal/pull/542))
+        - CLI reuse policy flag type ([#473](https://github.com/temporalio/temporal/pull/473))
+    - House cleaning:
+        - Remove executable permission from JSON files ([#483](https://github.com/temporalio/temporal/pull/483))
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:71]: I would remove this.

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+        - Describe Task ([#408](https://github.com/temporalio/temporal/pull/408), [#416](https://github.com/temporalio/temporal/pull/416), [#412](https://github.com/temporalio/temporal/pull/412)): `"shard describe_task --task_type [timer, replication, transfer] ..."` / `"dt"`
+        - List Cluster memberships([#423](https://github.com/temporalio/temporal/pull/423/files)): `"membership list_db"`
+        - List Ringpop members ([#426](https://github.com/temporalio/temporal/pull/426)): `"membership list_gossip"`
+        - List Replication DLQ Tasks ([#456](https://github.com/temporalio/temporal/pull/456)): `"kafka list_dlq"`
+    - Admin Command improvements:
+        - Added filtering of Tasks from TaskQueue based on Workflow and run Ids  ([#462](https://github.com/temporalio/temporal/pull/462))
+        - Consistency to CLI admin command names ([#427](https://github.com/temporalio/temporal/pull/427))
+        - Parameter types changed from enum to string ([#445](https://github.com/temporalio/temporal/pull/445), [#447](https://github.com/temporalio/temporal/pull/447))
+        - Table-view support for CLI list commands ([#444](https://github.com/temporalio/temporal/pull/444))
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:64]:
```suggestion
        - Table-view support for list commands ([#444](https://github.com/temporalio/temporal/pull/444))
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+        - Admin cluster command for adding search attributes to be indexed ([#514](https://github.com/temporalio/temporal/pull/514))
+        - Non-string heartbeat details fail to display and only the first heartbeat detail shows up in CLI ([#525](https://github.com/temporalio/temporal/pull/525))
+        - Printing Query CLI command results ([#542](https://github.com/temporalio/temporal/pull/542))
+        - CLI reuse policy flag type ([#473](https://github.com/temporalio/temporal/pull/473))
+    - House cleaning:
+        - Remove executable permission from JSON files ([#483](https://github.com/temporalio/temporal/pull/483))
+- **Error chaining** ⛓️  :  Errors can now be chained together across SDKs in different languages and structured as Protobuf messages.
+- **Simplified SQL Schemas** ➰ :
+- **All Queries are strongly consistent** 🔎 : The actual current state is always returned.
+- **Protobuf changes** 👣 : All payloads (Workflow input, Activity input, etc...) sent to Temporal now have headers and data fields.
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:75]: Let's move it to protobuf section on top:
```
- All payloads (Workflow input, Activity input, Error details, etc...) sent to Temporal now have metadata and data fields.
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 15]
```diff
@@ -0,0 +1,111 @@
...
+        - Parameter types changed from enum to string ([#445](https://github.com/temporalio/temporal/pull/445), [#447](https://github.com/temporalio/temporal/pull/447))
+        - Table-view support for CLI list commands ([#444](https://github.com/temporalio/temporal/pull/444))
+    - Fixes
+        - Admin cluster command for adding search attributes to be indexed ([#514](https://github.com/temporalio/temporal/pull/514))
+        - Non-string heartbeat details fail to display and only the first heartbeat detail shows up in CLI ([#525](https://github.com/temporalio/temporal/pull/525))
+        - Printing Query CLI command results ([#542](https://github.com/temporalio/temporal/pull/542))
+        - CLI reuse policy flag type ([#473](https://github.com/temporalio/temporal/pull/473))
+    - House cleaning:
+        - Remove executable permission from JSON files ([#483](https://github.com/temporalio/temporal/pull/483))
+- **Error chaining** ⛓️  :  Errors can now be chained together across SDKs in different languages and structured as Protobuf messages.
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:72]: Add:
- Errors are serialized using special proto type `Failure` and can be transmitted between different languages.

Maybe this is also for protobuf section on top.

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 16]
```diff
@@ -0,0 +1,111 @@
...
+- **Simplified SQL Schemas** ➰ :
+- **All Queries are strongly consistent** 🔎 : The actual current state is always returned.
+- **Protobuf changes** 👣 : All payloads (Workflow input, Activity input, etc...) sent to Temporal now have headers and data fields.
+- **SDKs** 🔌 : Improved experience for both Java SDK and Go SDK.
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:76]:
```suggestion
- **SDKs** 🔌 : Significantly improved experience for both Java SDK and Go SDK.
```
:-)

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 17]
```diff
@@ -0,0 +1,111 @@
...
+- **When using Docker** 🐳 :
+    - `BroadcastAddress` can be configured via the `TEMPORAL_BROADCAST_ADDRESS` environment variable.
+    - `MYSQL_TX_ISOLATION_COMPAT` environment variable can be set as `true` in order to support setting up schemas with pre-5.7.20 MySQL installations.
+    - Simplified getting started experience to have Docker pre baked with 'default' namespace.
+- **Other minor changes** 🔹 :
+    - Pass 127.0.0.1 explicitly to Cassandra in dependency docker ([#540](https://github.com/temporalio/temporal/pull/540))
+    - Remove statsd from / add admin-tools to docker-compose files ([#536](https://github.com/temporalio/temporal/pull/536))
+    - Expose cassandra consistency and serial consistency settings in configuration ([#533](https://github.com/temporalio/temporal/pull/533))
+    - Adopt Google api-linter ([#526](https://github.com/temporalio/temporal/pull/526))
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:85]:
```suggestion
    - Adopt "Google api-linter" and "Buf" linters ([#526](https://github.com/temporalio/temporal/pull/526))
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 18]
```diff
@@ -0,0 +1,111 @@
...
+    - NamespaceMetadata Partitioning Alignment ([#486](https://github.com/temporalio/temporal/pull/486))
+    - Upgrade gocql to latest release ([#498](https://github.com/temporalio/temporal/pull/498))
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:87]: Remove.

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 19]
```diff
@@ -0,0 +1,111 @@
...
+    - Upgrade gocql to latest release ([#498](https://github.com/temporalio/temporal/pull/498))
+    - Support Dynamic Cluster IP Addresses in Failure Scenarios ([#495](https://github.com/temporalio/temporal/pull/495))
+    - Add GOBIN to PATH in Makefile ([#485](https://github.com/temporalio/temporal/pull/485))
+    - Fix retry for wrapped failures ([#490](https://github.com/temporalio/temporal/pull/490))
+    - Tighten Ringpop Join Params ([#446](https://github.com/temporalio/temporal/pull/446))
+    - Set ParentClosePolicy default value ([#481](https://github.com/temporalio/temporal/pull/481))
+    - Use Failure and FailureInfo for errors ([#397](https://github.com/temporalio/temporal/pull/397))
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:94]: Remove.

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 20]
```diff
@@ -0,0 +1,111 @@
...
+    - Save last RetryStatus for retryable failures ([#432](https://github.com/temporalio/temporal/pull/432))
+    - Set proper timeout for Workflows ([#398](https://github.com/temporalio/temporal/pull/398))
+    - Prefix enums with type and convert to upper case ([#440](https://github.com/temporalio/temporal/pull/440))
+    - Allow for empty postgres db password ([#461](https://github.com/temporalio/temporal/pull/461))
+    - Remove query consistency level from request ([#452](https://github.com/temporalio/temporal/pull/452))
+    - Add UNSPECIFIED to every enum ([#441](https://github.com/temporalio/temporal/pull/441))
+    - Call cancel on context to prevent memory leak ([#438](https://github.com/temporalio/temporal/pull/438))
+    - Align dynamic config with proto enum ([#417](https://github.com/temporalio/temporal/pull/417))
+    - Emit domain specific metrics in Task processing
+    - Remove Workflow timeout type ([#434](https://github.com/temporalio/temporal/pull/434))
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:104]: Remove.

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 21]
```diff
@@ -0,0 +1,111 @@
...
+<!--truncate-->
+
+Today is an exciting day for us! Temporal is releasing its newest version, v0.28.0, bringing us one step closer to a stable production ready release. We thank all of the users who have helped us iterate and move the product forward.
+
+This changelog includes all of the changes since Temporal forked from Cadence, and should help users who are interested in upgrading.
+
+## What has changed?
+
+- **Cadence Oct 2019** → **Temporal v0.28.0 Alpha**: The software version has bumped.
+- **Thrift/TChannel** → **Protobug/gRPC**:
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:24]:
```suggestion
- **Thrift/TChannel** → **[Protobuf](https://developers.google.com/protocol-buffers)/[gRPC](https://grpc.io/)**:
```

Comment (alexshtin) [reply blog/2020-07-31-temporal-v0.28.0-changelog.md:24]: Hopefully it will look good.

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 22]
```diff
@@ -0,0 +1,111 @@
...
+    - Server Thrift API → Protobuf based Go API (go.temporal.io/api).
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:25]:
```suggestion
    - Server Thrift API → Protobuf [API](https://github.com/temporalio/api). Go import path is `go.temporal.io/api`.
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 23]
```diff
@@ -0,0 +1,111 @@
...
+    - All communication over the wire can be encrypted via [TLS](https://docs.temporal.io/docs/configuring-temporal-server#tls). Note: Support for TLS in `tctl` and `temporal-web` is still in-progress.
+    - Thrift errors have been converted to custom service errors backed by standard [gRPC error codes](https://pkg.go.dev/google.golang.org/grpc/codes).
+    - Standard gRPC health service is used for health checks.
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:28]:
```suggestion
    - Standard [gRPC health service](https://github.com/grpc/grpc/blob/master/doc/health-checking.md) is used for health checks.
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 24]
```diff
@@ -0,0 +1,111 @@
...
+- **Workflow timeout refactors** ⌛:
+    - Workflow timeouts are optional.
+    - Workflow Executions have 3 timeouts available:
+        - Renamed:`ExecutionStartToCloseTimeout` → `WorkflowRunTimeout`.
+        - `WorkflowExecutionTimeout` is new. It limits the total Workflow execution time including retries and continue-as-new. It also enables limiting the total time of a cron job. The default is 10 years. `WorkflowRunTimeout` defaults to the value of `WorkflowExecutionTimeout`.
+        - Renamed:`DecisionTaskStartToCloseTimeout` → `WorkflowTaskTimeout`. Defaults to 10 seconds.
+- **Workflow retry refactors** 🔁 :
+    - `ExpirationInterval` has been removed from `RetryPolicy`. Instead, `WorkflowExecutionTimeout` and `ScheduleToClose` are used to stop retries for Workflows and Activities respectively.
+- **Activity retry refactors** 🔁 :
+    - Temporal includes default `RetryOptions` for Activities.
```
Comment (mfateev) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:49]: May be include information about the defaults: 
```
initialInterval = 1 second
maximumInterval = 100 * initialInterval
backoffCoefficient = 2
```

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 25]
```diff
@@ -0,0 +1,111 @@
...
+    - To disable Activity retries, configure Activities with `RetryOptions` that set `maximumAttempts` to 1.
+- **tctl (CLI) updates** 🧰 :
+    - New admin commands:
```
Comment (mfateev) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:52]: Maybe prepend the sample commands with admin where appropriate? "admin shard describe" etc?

File: blog/2020-07-31-temporal-v0.28.0-changelog.md [hunk 26]
```diff
@@ -0,0 +1,111 @@
...
+- **Helm charts** ☸️  : [Deploy Temporal server to a Kubernetes Cluster](https://github.com/temporalio/helm-charts).
+- **Entity renames** 🏷️ :
+    - `Decision` → `Command`
+    - `DecisionTask` → `WorkflowTask`
+    - `Decider` → `Workflow`
+    - `Domain` → `Namespace`
+    - `NamespaceStatus` → `NamespaceState`
+    - `TaskList` → `TaskQueue`
+    - `ArchivalStatus` → `ArchivalState`
+- **Import path rename** 📦 : go.uber.org/cadence → go.temporal.io/sdk
```
Comment (alexshtin) [inline blog/2020-07-31-temporal-v0.28.0-changelog.md:39]:
```suggestion
- **Import path rename** 📦 : `go.uber.org/cadence` → `go.temporal.io/sdk`
```
